sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -84,46 +76,77 @@ async function main() {
     throw appComponentCreationResult.body.error;
   }
 
-  // Creating the test run
-  const testRunCreationResult = await client.path("/testruns/{testRunId}", testRunId).patch({
-    contentType: "application/merge-patch+json",
-    body: {
-      testId: testCreationResult.body.testId,
-      displayName: displayName,
-      vusers: 10,
-    },
-  });
+  // Checking the test run status and printing metrics
+  var testRunResult = await (await beginCreateOrUpdateTestRun(client, testId, displayName)).pollUntilDone();
+
+  if (isUnexpected(testRunResult)) throw new Error("There is some issue in running the test.");
+
+  var getTestRunResult = await client.path("/test-runs/{testRunId}", testRunResult.body.testRunId).get();
```

Comment:
Is not this the same thing as `testRunResult`?

Reply:
Yes, removed the getTestRunResult.